js/sparkswap.js

```diff
@@ -317,7 +317,7 @@ module.exports = class sparkswap extends Exchange {
         }
         const response = {
             'info': order,
-            'id': id,
+            'id': order.order_id,
```

Comment:
Oops, is has to be `order['order_id']` as explained here: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#accessing-dictionary-keys

Reply:
@kroitor will revise when going function by function (there are a ton more issues like this)

Reply:
@dannypaz thx so much!